src/components/Game/StateForGame.tsx

```diff
@@ -0,0 +1,17 @@
+import { useState } from 'react';
+import { Game } from './Game';
+import { PokemonsData } from '../../types/PokemonsData';
+
+type Props = {
+  pokemonPlayer: PokemonsData | undefined;
+};
+
+export const StateForGame = ({ pokemonPlayer }: Props): JSX.Element => {
```

Comment:
I don't see a need in such a wrapper component that just passes values to `Game`